msg/PropAngleRange.msg

```diff
@@ -1,3 +1,3 @@
 string prop_label
-float32 theta_1
-float32 theta_2
\ No newline at end of file
+float64 theta_1
```

Comment:
idrk but is ```theta_1``` descriptive enough?

Reply:
I thought about it and I don't want to do theta_left and theta_right because the LiDAR range is larger than 180 and depending on where an object is both angles could be to the left or to the right. Going to leave as theta_1 and theta_2 but I'll add comments to describe them.

Reply:
Nvm, changing to theta_small and theta_large